Npoi.Mapper/src/Npoi.Mapper/Mapper.cs

```diff
@@ -553,15 +568,15 @@ public void Save<T>(Stream stream, int sheetIndex = 0, bool overwrite = true, bo
 
         private Type GetDynamicType(ISheet sheet)
         {
-            var firstRowIndex = sheet.FirstRowNum;
+            var firstRowIndex = sheet.FirstRowNum+ HeaderRowIndex;
```

Comment:
`HeaderRowIndex` is the way to specify header row index, I don't we can use it as skipped count. 

<a id="51fb5c80-feba-11e8-af3b-27dc92f5defc" href="none" codereview-uuid-reserved-tag />